src/content/docs/infrastructure/elastic-container-service-integration/monitor-ecs-with-apm-agents.mdx

```diff
@@ -0,0 +1,280 @@
+                },
+                # License key is required
+                {
+                    "name": "NEW_RELIC_LICENSE_KEY",
+                    "value": "your-license-key"
+                }
+            ],
+          ```
+        </TabsPageItem>
+
+        <TabsPageItem id="nodejs">
+          ```task-definition
+            "environment": [
+                # Specify your region-aware host here
+                {
+                    "name": "NEW_RELIC_HOST",
+                    "value": "collector.newrelic.com"
+                },
+                # App name is required
+                {
+                    "name": "NEW_RELIC_APP_NAME",
+                    "value": "Fargate Demo (AWS)"
+                },
+                # License key is required
+                {
+                    "name": "NEW_RELIC_LICENSE_KEY",
+                    "value": "your-license-key"
+                }
+            ],
+          ```
+        </TabsPageItem>
+
+        <TabsPageItem id="python">
+          ```task-definition
+            "environment": [
+                # Specify your region-aware host here
+                {
+                    "name": "NEW_RELIC_HOST",
+                    "value": "collector.newrelic.com"
+                },
+                # Setting an app name is highly recommended but not required
+                {
+                    "name": "NEW_RELIC_APP_NAME",
+                    "value": "Fargate Demo (AWS)"
+                },
+                # License key is required
+                {
+                    "name": "NEW_RELIC_LICENSE_KEY",
+                    "value": "your-license-key"
+                }
+            ],
+          ```
+        </TabsPageItem>
+
+      </TabsPages>
+    </Tabs>
+
+## Setting up the infrastructure agent as a sidecar [#infra-agent-side-car]
+You can also set up the New Relic [infrastructure agent](https://docs.newrelic.com/docs/infrastructure/install-infrastructure-agent/get-started/requirements-infrastructure-agent/) as a sidecar in addition to the language agent for more visibility into your environment.
+This will provide you with relationship mapping between APM and Infrastructure for features such as the service map and infra-specific metrics.
+
+<img
+  title="Screenshot showing service map when running language agent and infra agent together in Fargate environment"
+  alt="Service map screenshot running language agent and infra agent together in Fargate environment"
+  src="/images/service-map-screenshot-showing-language-and-infra-agent-in-fargate-env.png"
+/>
+
+<figcaption>
+  This is an example of how the service map would be populated if you run a language agent and the infrastructure agent together in a Fargate environment.
+</figcaption>
+
+For more information, please see our [ECS integration install guide](https://docs.newrelic.com/docs/infrastructure/elastic-container-service-integration/install-ecs-integration/).
+
+
+Example full task definition file:
```

Comment:
Can we add a code comment somewhere that identifies the changes specific to setting up a sidecar container?

Reply:
This task definition file was meant to show the full setup of the sidecar and the agent. I can add it under a new main heading to clarify that